public/config.js

```diff
@@ -2,8 +2,8 @@ import "https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/jsoned
 
 // GitHub config
 globalThis.ghConfig = {
-  githubRepo: "EOX-A/git-clerk-demo",
```

Comment:
Actually, I'd like to offer the EOX-A/git-clerk-demo repo as "official" demo repo to test against, so I'll revert this